packages/test-app/src/App.tsx

```diff
@@ -30,8 +30,8 @@ function App() {
   );
 
   useEffect(() => {
-    connection.db.player.onInsert(player => {
-      console.log(player);
+    connection.db.player.onInsert((ctx, player) => {
+      console.log(ctx, player);
```

Comment:
Was this fix made necessary by this PR or is it just a general fix?

Reply:
Looks like a general fix to me. Unrelated to this PR.